src/btree/bt_vrfy.c

```diff
@@ -527,8 +527,9 @@ __verify_addr_string(WT_SESSION_IMPL *session, WT_REF *ref, WT_ITEM *buf)
     WT_DECL_RET;
     char tp_string[2][WT_TP_STRING_SIZE];
 
+    WT_ERR(__wt_scr_alloc(session, 0, &tmp));
+
```

Comment:
This seems like it will fix the problem. Please also add an assert to the {{wt_addr_string}} function.